File: 2.0/python/contacts/createContactListFromFile.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.contacts.createContactListFromFile(
    name='My Contact List',
    file=open('contacts.csv', 'r')
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)

File: 2.0/php/contacts/createContactListFromFile.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->createContactListFromFile();
        $request->getOperationConfig()->setFileUpload(__dir__.'\contacts.csv', 'My Contact List');
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();
